recipes/cpp_project_framework/all/test_package/conanfile.py

```diff
@@ -0,0 +1,20 @@
+from conan import ConanFile
+from conan.tools.build import cross_building
+from conans import CMake
+import os
+
+
+class CppProjectFrameworkTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+    requires = "gtest/1.10.0"
```

Comment:
The idea is not testing the project, but the package. Create a very small and basic usage, a.k.a example.

Reply:
added basic examples in latest test package already

Reply:
I still don't see those changes

Reply:
https://github.com/conan-io/conan-center-index/blob/785e12c6a6062016f8ea7d7d2e26f39a515ac43d/recipes/cpp_project_framework/all/test_package/test_package.cpp#L7-L30

Reply:
I see, as your project does not generate any library, you need to generate a test. Makes sense.